configure.ac

```diff
@@ -1502,6 +1510,28 @@ AM_CONDITIONAL([SNMP], [test "x${SNMP_METHOD}" != "x"])
 AC_SUBST(SNMP_LIBS)
 AC_SUBST(SNMP_CFLAGS)
 
+dnl ---------------
+dnl libyang
+dnl ---------------
+PKG_CHECK_MODULES(libyang, [libyang >= 0.15.137], , [
```

Comment:
0.15.137 doesn't appear to be a released version of libyang.  0.15r1 which is the latest release on github, does not seem to build with frr.  Is there a released version of this package we can use?  I'm trying to get the libyang dependency into alpine linux (PR is here: https://github.com/alpinelinux/aports/pull/4922) and I _can_ use a git hash, but it would be preferable to use a released version...

Also, libyang doesn't appear to be packaged in debian or redhat yet, which makes it more difficult for users to build frr, are there plans to package libyang for debian or rpm?